src/js/view/captionsrenderer.js

```diff
@@ -237,11 +237,18 @@ const CaptionsRenderer = function (viewModel) {
 
             // Use screen dimensions when in fullscreen on mobile devices
             if (_model.get('fullscreen') && OS.mobile) {
-                // device is in portrait mode when window.orientation = 0 || 180
-                const portraitMode = !(window.orientation % 180);
                 const { screen } = window;
-                containerHeight = portraitMode ? screen.availHeight : screen.availWidth;
-                containerWidth = portraitMode ? screen.availWidth : screen.availHeight;
+                if (screen.orientation) {
```

Comment:
What if the orientation is 0? Seems like it'd be better to check if `orientation` is a number.

Reply:
`screen.orientation` is an object, not a number.